starter-core/src/main/java/io/micronaut/starter/feature/function/template/http/httpFunctionGroovyController.rocker.raw

```diff
@@ -0,0 +1,58 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName()
+}
+
+
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.annotation.Post
+import io.micronaut.http.annotation.Body
+import io.micronaut.http.annotation.Produces;
+import io.micronaut.http.MediaType;
+
+@@Controller
+class @project.getClassName()Controller {
+
+    @@Produces(MediaType.TEXT_PLAIN)
+    @@Get
+    String index() {
+        "Example Response"
+    }
+
+    @@Post
+    SampleReturnMessage postMethod(@@Body SampleInputMessage inputMessage){
+      SampleReturnMessage retMessage = new SampleReturnMessage()
+      retMessage.setReturnMessage("Hello "+inputMessage.getName()+", thank you for sending the message")
```

Comment:
We should use Groovy's string interpolation instead

Reply:
Moreover, in groovy instead of doing `inputMessage.getName()` you can just do `input.name`